### PR TITLE
Update db col names

### PR DESCRIPTION
**Pull request by blinker1104**

### CODE REVIEW

File: server/database/db.js [hunk 1]
```diff
@@ -9,12 +9,12 @@ const sequelize = new Sequelize('mydb', 'noobdev', 'post', {
 
 
 var biz = sequelize.define('business', {
-  "bizID" : Sequelize.TEXT,
+  "biz_iD" : Sequelize.TEXT,
```
Comment (nathanksou) [inline server/database/db.js:12]: Did you intend for this to be biz_iD? Or did you mean biz_id?

Comment (nathanksou) [reply server/database/db.js:12]: Also with these changes, do you need to change your seed.js to accommodate these new column headers?

Comment (blinker1104) [reply server/database/db.js:12]: oops, I will change that one. Thanks.

Comment (blinker1104) [reply server/database/db.js]: I don't have seed.js?

**Pull request merged**